ckanext/xloader/templates/xloader/resource_data.html

```diff
@@ -5,10 +5,26 @@
 {% block primary_content_inner %}
 
   {% set action = h.url_for('xloader.resource_data', id=pkg.name, resource_id=res.id) %}
+  {% set delete_action = h.url_for('xloader.delete_datastore_table', id=pkg.id, resource_id=res.id) %}
   {% set show_table = true %}
 
+  {% block delete_ds_button %}
+    {% if res.datastore_active %}
+      <form method="post" action="{{ delete_action }}" class="mb-3 d-inline-block">
+        {{ h.csrf_input() if 'csrf_input' in h }}
+        <a href="{{ h.url_for('xloader.delete_datastore_table', id=pkg.id, resource_id=res.id) }}"
+          class="btn btn-danger pull-left"
```

Comment:
This should use 'pull-right' so the upload button comes first.

Reply:
should stop using pulls in CKAN really. They truly are the worst thing for CSS layout and ordering like this. I can wrap in flex box and do a flex revers is probably best

Reply:
@ThrawnCA okay this should be good not